libecho/src/Packet.cc

```diff
@@ -0,0 +1,117 @@
+#include "Packet.h"
+
+// #define crcpp_uint32
+#include <arpa/inet.h>
```

Comment:
to unixowy nagłówek, jak będziemy się przenosić na windowsa to może się nie skompilować

Reply:
Jaki jest windowsowy odpowiednik?

Reply:
@a-piasta

Reply:
winsock2.h